A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js

```diff
@@ -1006,6 +1006,40 @@ var A320_Neo_UpperECAM;
                             },
                         ]
                     },
+                    {
+                        name: "APU",
+                        messages: [
+                            {
+                                message: "AUTO SHUT DOWN",
+                                level: 2,
+                                flightPhasesInhib: [3, 4, 5, 7, 8],
+                                isActive: () => this.getCachedSimVar("L:A32NX_APU_IS_AUTO_SHUTDOWN", "Bool"),
```

Comment:
I assume the APU code already enforces this correctly, but you could ensure that A32NX_APU_IS_EMERGENCY_SHUTDOWN is really false here so only one warning is shown here, even if both simvars are set.

Reply:
Yes, the APU code handles this correctly. I'd like to avoid redeclaring that here, as to ensure we don't duplicate that logic. 

I'm sure that in the future there are other such cases where one message should hide another. Handling this should probably be done by the Rust FWC once it exists. Shall we revisit this when we have some Rust FWC code?